control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -1890,6 +1896,40 @@ func (r *HostedControlPlaneReconciler) reconcileClusterVersionOperator(ctx conte
 	return nil
 }
 
+func (r *HostedControlPlaneReconciler) reconcileClusterNetworkOperator(ctx context.Context, hcp *hyperv1.HostedControlPlane, releaseImage *releaseinfo.ReleaseImage) error {
```

Comment:
Is it correct that we always do this regardless of the CNI type?

Reply:
Even when CNO is deployed in the management cluster it manages the hosted cluster by default, it shouldn't be an issue.